spec/unit/block_score/person_spec.rb

```diff
@@ -29,22 +31,23 @@
     it { is_expected.to be_persisted }
     its(:name_first) { is_expected.not_to be_empty }
     it { is_expected.to be_an_instance_of described_class }
-    context 'null id presented' do
-      let(:person_id) { nil }
 
-      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
-    end
-
-    context 'empty id presented' do
-      let(:person_id) { '' }
-
-      it { expect { described_class.retrieve(person_id) }.to raise_error(ArgumentError, 'ID must be supplied') }
+    context 'no id presented' do
+      it 'has to have some ID' do
```

Comment:
I think "has to have" is a bit clumsy, what about: `it 'must have a valid id'` ?